llvm/include/llvm/IR/ConstantRangeList.h

```diff
@@ -72,6 +72,14 @@ class [[nodiscard]] ConstantRangeList {
                          APInt(64, Upper, /*isSigned=*/true)));
   }
 
+  /// Return the range list that results from the union of this range
```

Comment:
small nit: "of this range *list* with another..." or "of this list with another" ?

Reply:
Done.